hs_core/views/resource_rest_api.py

```diff
@@ -768,8 +768,10 @@ def list(self, request, *args, **kwargs):
                 try:
                     resource_file_info_list.append(self.resourceFileToListItem(f))
                 except SessionException as err:
-                    # primarily this exception will be raised if the file is not found in iRODS
                     logger.error(f"Error for file {f.storage_path} from iRODS: {err.stderr}")
+                except Exception as err:
+                    # primarily this exception will be raised if the file is not found in iRODS
+                    logger.error(f"Error for file {f.storage_path} from iRODS: {str(err)}")
```

Comment:
The use of `f.storage_path `results in 1 db query. If we want we should be able to avoid it.

Reply:
Got it, that makes sense.
Why not target `ValidationError` specifically instead of general `Exception` ?

Reply:
Fixed it.